freedata_gui/src/components/dynamic_components.vue

```diff
@@ -524,6 +593,15 @@ onMounted(() => {
           Fill grid with common widgets
         </button>
       </div>
+      <div>
+        <button
+          class="btn btn-sm btn-outline-primary"
```

Comment:
can we add a "mt-1" or "mt-2" here for having some space between both buttons?

Reply:
Will update this evening.